### DISCUSSION TITLE
🎉 It´s time for what? Graduation! Share yourselves! 👨🏻‍🎓👩🏿‍🎓

### DISCUSSION THREAD

**Discussion by Guedesou** [community/community#17744] [GitHub Education] [answered]
<h1 align="center"> 🎊Like that song: I´m Happier Than Ever 🎊</h1></p>
<h4 align="center">I'm very excited about graduation this year. </br>Being part of this community, having support from colleagues and teachers was a watershed for me. </br>
While we wait for the event to start <a href="https://github.com/Guedesou/GitHubGraduation-2022">(if you don't know what I'm talking about, click here)</a>... </br> ✨ How about we do a little networking? ✨
  </h4>

<h4 align="center">Share your GitHub profile! I will follow everyone who follows me there. ☜(ﾟヮﾟ☜)</br>
Let's follow the new Devs and  changers who impacts around the world.</br> So happy to be part of this moment with you!</br>
<a href="https://github.com/Guedesou">Follow me HERE</a></h4>

**Comment by mlarasusan** [comment]
Yay, congratulations on your Graduation @Guedesou [💖](https://emojipedia.org/sparkling-heart/) So happy to have you joining us on June 11th 🎉

**Comment by samus-aran** [comment, marked as answer]
Awesome stuff @Guedesou! I hope you got a chance to meet others in [Class 2022](https://education.github.com/graduation/yearbook) 🎉  As we have now completed Class 2022 graduation we will lock this conversation and good luck on your future learning or adventures 😄
